### PR TITLE
[Docs] general display fixes + remove placeholder/tmp content

### PR DESCRIPTION
**Pull request by pedro-psb**
**Needs squash after approval**

What this does:

1. Replace placeholder Getting Started with page from pulpproject  (and update it)
    * **To review the changes from the original, see the `Update 'Start Here' page` commit**
3. Fix image links in `admin/learn/rbac` and rename overview->index
4. Remove "Triage-needed" tmp dirs and un-nest its content
5. Fix diagram links in `user/learn/concepts`
6. Fix image link in `admin/lean/architecture`
7. Move "Settings" and "Tech-preview" to "Reference" section
8. Use new `_SUMMARY.md` file for ordering and fix numbered filenames

### 1. New "Getting Started"

image

### 2. Fix images in RBAC + add index page

image

### 3. Remove Triage Needed from dev section

image

### 4. Add diagram images to User > Concepts

image

### 5. Add Architecture image

image

### 6. Move Settings/tech-preview to Reference

image

### 7. Use new ordering mechanism

image

### CODE REVIEW

File: staging_docs/user/tutorials/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Start Here
```
Comment (bmbouter) [inline staging_docs/user/tutorials/index.md:1]: These are great!

**Review by bmbouter — APPROVED** [review]
I think these docs are great. Thanks!

**Review by decko — DISMISSED** [review]

File: staging_docs/dev/learn/rbac/index.md [hunk 2]
```diff
@@ -11,28 +11,23 @@ The goals of the authorization system are to:
...
+  etc. See the :ref:`permission_checking_machinery` section for the complete list of available
+  methods.
 
-:Permission Checking Machinery: A set of methods which can check various conditions such as if a
-    requesting user has a given permission, or is a member of a group that has a given permission,
-    etc. See the :ref:`permission_checking_machinery` section for the complete list of available
-    methods.
+Users and Groups
+: Users and Groups live in the Django database and are used by the Permission Checking Machinery.
+  See the :ref:`users_and_groups` documentation for more information.
```
Comment (lubosmj) [inline staging_docs/dev/learn/rbac/index.md:29]: We miss the label for these two references:
image
image

File: staging_docs/user/tutorials/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+If you're a new user and are unsure where to begin, this page outlines the different options available, as well as limitations and requirements for those options.
```
Comment (lubosmj) [inline staging_docs/user/tutorials/index.md:3]:
```suggestion
If you are a new user and are unsure where to begin, this page outlines the different options available, as well as the limitations and requirements for those options.
```

**Comment by pedro-psb** [comment]
@lubosmj I started looking into these links issues that you pointed out, and there is really A LOT of similar links issues caused by the rst conversion (most notably on the dev section). Can I open a separate issue for this? I think its outside the scope of this PR.

I'm removing the "Triage Needed" dirs not because they are now triaged, but because I think it wasnt a very effective idea and its polluting the navigation.

**Review by lubosmj — APPROVED** [review]

**Review by decko — APPROVED** [review]

**Pull request merged**